eng/test-rebuild.ps1

```diff
@@ -15,6 +16,7 @@ function Print-Usage() {
   Write-Host "Usage: test-rebuild.ps1"
   Write-Host "  -configuration            Build configuration ('Debug' or 'Release')"
   Write-Host "  -ci                       Set when running on CI server"
+  Write-Host "  -noBuild                  If set, skips building before running the rebuild"
```

Comment:
Believe this impacts the bootstrap build only correct? If so can we specify that?

Reply:
```suggestion
  Write-Host "  -noBuild                  If set, skips running a bootstrap build before running the rebuild"
```